langlib/lang.float/src/main/ballerina/float.bal

```diff
@@ -55,6 +65,11 @@ public isolated function isInfinite(float x) returns boolean = @java:Method {
 #
 # Exactly one of isFinite, isInfinite and IsNaN will be true for any float value.
 #
+# ```ballerina
+# float nanValue = 0.0/0.0;
```

Comment:
We can wrap the `0.0/0.0` to another variable, or we can use ballerina `float:NaN`
what will be the best choice?

Reply:
I prefer `float:NaN` in order to keep the example simple. See https://github.com/ballerina-platform/ballerina-lang/pull/38905#discussion_r1037775654 for variable naming though.

Reply:
+1 to use `float:NaN`

Reply:
I am also +1 to float:NaN .

Reply:
Updated